frontend/io-webapp/src/session/Question.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+
+const Question = (props) => {
+  const sampleQuestions = [
+    {
+      id: 1,
+      question: "What's the first letter of the alphabet?",
+      answers: ["A", "D", "F", "P"],
+    },
+    {
+      id: 2,
+      question: "What's the capital of Germany?",
+    },
+  ];
+
+  const [questions, setQuestions] = React.useState(sampleQuestions);
+  const [current, setCurrent] = React.useState(questions[0]);
+
+  const submitHandler = (id) => {
+    /* TODO */
+    const newQuestions = questions;
+    newQuestions.shift();
+    console.log(newQuestions);
+    setQuestions(newQuestions);
+    setCurrent(questions[0]);
+  };
+
+  return (
+    <div className="question" style={{ width: props.width }}>
+      {questions.length ? (
+        current.hasOwnProperty("answers") ? (
+          <div key={current.id} className="specificQuestion">
+            <h1 className="questionProper">{current.question}</h1>
+            <div className="questionRow">
+              <div className="answer">
+                <button type="button" onClick={() => submitHandler()}>
+                  A
+                </button>
+                <p>{current.answers[0]}</p>
+              </div>
+              <div className="answer">
+                <button type="button" onClick={() => submitHandler()}>
+                  B
+                </button>
+                <p>{current.answers[1]}</p>
+              </div>
+            </div>
```

Comment:
Wizualnie mam wrażenie, że sporo się z tego dubluje ponizej. Może warto by było tutaj wydzielić jeszcze jakiś jeden komponent?

Reply:
Ten cały kod związany z odpowiedziami ABCD jest bardzo brzydki i chaotyczny i zdecydowanie do przepisania, nie wiem jeszcze do końca jak, ale wcześniej czy później to poprawię.